source/gcp/index.rst

```diff
@@ -46,3 +46,12 @@ The data flow between the Wazuh module and Cloud Pub/Sub looks as follows:
 .. thumbnail:: ../images/gcp/gcp-data-flow.png
     :align: center
     :width: 100%
+
+Performance
+-----------
+
+The Wazuh module for GCP Pub/Sub uses a `Python library <https://googleapis.dev/python/pubsub/2.7.1/index.html/>`_ to pull the messages from the topic. Its implementation has design limitations when integrated with the Wazuh module.
+
+To improve the integration performance, it is highly recommended to focus on horizontal escalation, configuring this module for the same subscription on different machines (managers or agents).
```

Comment:
```suggestion
To improve the integration performance, it is highly recommended to use horizontal scalability, configuring this module for the same subscription on different machines (managers or agents).
```

Reply:
Solved mistake in 79cc10a17da0d8a90535cb7ada0dfa97168d8426.